build.gradle

```diff
@@ -1,5 +1,6 @@
 buildscript {
     repositories {
+        google()
```

Comment:
In all these PRs put the google repo last, after all the existing ones

Reply:
done